Makefile

```diff
@@ -118,6 +120,8 @@ build/llvm.BUILT:
 		install-objdump \
 		install-objcopy \
 		install-c++filt \
+		install-LLVM \
+		install-clang-cpp \
```

Comment:
What does this line install?  `cpp` I assume refers to the pre-processor rather than clang++?

Reply:
I commented a bit more above on this, but I thought `clang-cpp` was the preprocessor as well. Apparently though this deals with `libclang-cpp.so` and may just be slightly-oddly-named.